reactioncommerce/reaction
# Show products that fail validation when publishing.

Resolves issue #2690 

Previously we could identify which products are missing the label when publishing, the red button shows which products are missing the label.

**How to Test**

1. Login as Admin
2. Create a product but leave "Label" blank
3. Navigate to the product grid
4. Hit "Publish All"
5. Observe that the products don't publish, and there is a red button as an indicator of which product it is that needs changes. 
6. When the problem product is opened, observe that the `edit variant` panel is also opened and the label field flashes red to notify that the label is missing.
